multy_core/src/bitcoin/bitcoin_transaction.h

```diff
@@ -46,11 +46,11 @@ class BitcoinTransaction : public TransactionBase
     void sign();
 
     template <typename T>
-    void serialize_to_stream(T* stream) const;
+    void serialize_to_stream(T* stream, bool with_change_destinations) const;
```

Comment:
Don't use `bool`-parameters, please add an enum with verbose and descriptive values.